src/dotnet/Microsoft.DotNet.Core.Sdk.targets

```diff
@@ -20,10 +20,11 @@
 
   <PropertyGroup>
     <GenerateRuntimeConfigurationFiles Condition=" '$(GenerateRuntimeConfigurationFiles)' == '' and '$(OutputType)' == 'exe' ">true</GenerateRuntimeConfigurationFiles>
+    <Version Condition=" '$(Version)' == '' ">1.0.0</Version>
```

Comment:
Can we set a default $(VersionPrefix) and $(VersionSuffix) and have this value be $(VersionPrefix)$(VersionSuffix) to align with migrations?